src/transport_picoquic.cpp

```diff
@@ -592,6 +594,18 @@ PicoQuicTransport::start()
     return cid;
 }
 
+void PicoQuicTransport::pq_runner() {
+
+    while (picoquic_runner_queue.size() > 0) {
+        auto cb = std::move(picoquic_runner_queue.pop());
+        if (cb) {
+            (*cb)();
+        } else {
+            break;
+        }
+    }
```

Comment:
Could this be achieved the same way by doing:
```cpp
do {
    if(auto cb = std::move(picoquic_runner_queue.pop())) (*cb)();
} while (picoquic_runner_queue.size() > 0);
```
?

Reply:
Or, even just omitting the break, since if it is null then nothing will happen, then it will try to loop, hit the condition, and then break the loop anyways.

Reply:
Yep, I can update it.